exercises/practice/list-ops/.docs/instructions.append.md

```diff
@@ -0,0 +1,100 @@
+# placeholder
+
+## New AWK concepts
+
+This is the first exercise where the solution is not a "main" script.
+The goal is to write a library of functions to be included into other awk programs.
+There are several interesting new concepts to see in this exercise.
+
+The following sections that are tagged with "(gawk)" are specifically GNU awk extensions.
+
+### Include files <sup>(gawk)</sup>
+
+Notice the [`@include`][d-include] directive in the test
+file.  This instructs gawk to read and evaluate the named file.
+
+### Namespaces <sup>(gawk)</sup>
+
+All awk variables are global
```

Comment:
The downside of fixed-width text :) If you make edits, you need to reformat. Or live with unbalanced blocks -- which I'm totally fine with.

Reply:
I know. This one was in-between: I intended to make the review easier